app/models/manageiq/providers/amazon/cloud_manager/provision_workflow.rb

```diff
@@ -47,16 +47,16 @@ def allowed_availability_zones(_options = {})
     allowed_ci(:availability_zones, [:cloud_network, :cloud_subnet, :security_group], targets.map(&:id))
   end
 
+  def self.provider_model
+    ManageIQ::Providers::Amazon::CloudManager
+  end
+
```

Comment:
@djberg96 can you tell if this was intended to be private?  If so I'd prefer either using `private_class_method` or `class << self; private` over making it public

Reply:
Based on https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_provision_virt_workflow.rb#L361-L365 I would say it was meant to be public.